lib/solution_tester.rb

```diff
@@ -73,7 +73,7 @@ def dockerfile_path
   end
 
   def solution_code_dir_for_stage(stage)
-    "#{solutions_dir}/#{@language.slug}/#{stage.slug}/code"
+    @course.stage_path(@language, stage, "code")
```

Comment:
```suggestion
    "#{stage.solution_dir}/code"
```

(i.e. no backward compatibility)